requirements.txt

```diff
@@ -5,15 +5,15 @@ Jinja2==3.1.3
 joblib==1.3.2
 livereload==2.6.3
 lunr==0.7.0
-Markdown==3.5.2
+Markdown==3.8.*
 MarkupSafe==2.1.3
-mkdocs==1.5.3
+mkdocs==1.6.1
```

Comment:
Why the bump for mkdocs? We should keep all packages the same and this is the reference

Reply:
Ok changed back.